adapters/iqzone/params_test.go

```diff
@@ -0,0 +1,42 @@
+package iqzone
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json schema. %v", err)
+	}
+
+	for _, p := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderIQZone, json.RawMessage(p)); err != nil {
+			t.Errorf("Schema rejected valid params: %s", p)
+		}
+	}
+}
+
+func TestInvalidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json schema. %v", err)
+	}
+
+	for _, p := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderIQZone, json.RawMessage(p)); err == nil {
+			t.Errorf("Schema allowed invalid params: %s", p)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"placementId": "test"}`,
+}
+
+var invalidParams = []string{
+	`{"placementId": 42}`,
```

Comment:
If you decide to keep the `"placementId"` listed in the `"required"` array inside `static/bidder-params/iqzone.json`, can we please add a couple more test cases in order to assert the request will be invalid when `"placementId"` isn't found in the `Ext`?
```
40   var invalidParams = []string{
41       `{"placementId": 42}`,
   +     `{}`,
   +     `{"someOtherParam": "value"}`,
42   }
adapters/iqzone/params_test.go
```